doc/releases/changelog-0.29.0.md

```diff
@@ -313,6 +312,8 @@
            [0.92, 1.  , 0.91, 1.  ]]], requires_grad=True)
   ```
 
+  Note that we have extracted the first probability vector entry for each 100 shot evaluation.
```

Comment:
```suggestion
  Note that we have extracted the first probability vector entry for each 100 shot evaluations.
```

Reply:
:thinking: For me the non plural reads correctly.

Reply:
"for each evaluation".  Think the singular is right.